openapi.yaml

```diff
@@ -143,374 +82,417 @@ paths:
+            $ref: '#/definitions/MealPlanSetResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/meal-plans/{mealPlanId}:
     get:
       description: Get a specific meal plan for a person
       tags:
         - meal-plans
-      operationId: getMealPlanByID
+      operationId: getMealPlanById
       parameters:
-        - in: path
-          name: mealPlanID
-          description: ID of meal plan
-          required: true
-          type: string
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/osuId'
+        - $ref: '#/parameters/mealPlanId'
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/MealPlanResultObject'
-        404:
-          description: Person or meal plan not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/addresses:
+            $ref: '#/definitions/MealPlanResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/address:
     get:
       description: Get addresses for a person
       tags:
         - addresses
       operationId: getAddresses
       parameters:
+        - $ref: '#/parameters/osuId'
+        - name: city
+          in: query
+          type: string
+          required: false
+          description: Filter results by city.
+        - name: county
+          in: query
+          type: string
+          required: false
+          description: Filter results by county.
+        - name: state
+          in: query
+          type: string
+          required: false
+          description: Filter results by state.
+        - name: stateCode
+          in: query
+          type: string
+          required: false
+          description: Filter results by state code.
+        - name: nation
+          in: query
+          type: string
+          required: false
+          description: Filter results by nation.
+        - name: nationCode
+          in: query
+          type: string
+          required: false
+          description: Filter results by nation code.
         - $ref: '#/parameters/addressType'
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/AddressesResultObject'
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
+            $ref: '#/definitions/AddressSetResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
     post:
       description: Create address for a person
       tags:
         - addresses
       operationId: postAddresses
```

Comment:
For consistency, it should be `postAddress` since I think we can only create one address at a time.